src/Tribe/Main.php

```diff
@@ -4610,5 +4557,225 @@ public function set_meta_factory_global() {
+		 * If it's not, it loads the Tickets framework from our
+		 * vendor/ submodule.
+		 */
+		public function maybe_load_tickets_framework() {
+			if ( defined( 'TRIBE_TICKETS_DIR' ) ) {
+				return;
+			}
+
+			// Give the standalone plugin a chance to load on activation
+			// WordPress loads all the active plugins before activating a new one.
+			if ( isset( $_GET['action'] ) && $_GET['action'] == 'activate' && isset( $_GET['plugin'] ) && strstr( $_GET['plugin'], 'tickets.php' ) ) {
+				return;
+			}
+
+			require_once $this->plugin_path . 'vendor/tickets/tickets.php';
+		}
+
+		/**
+		 * Inject TEC specific setting fields into the general tab
+		 *
+		 * @param array $general_tab_fields Fields for the general settings tab
+		 */
+		public function general_settings_tab_fields( $general_tab_fields ) {
+			require_once $this->plugin_path . 'src/admin-views/tribe-options-general.php';
+
+			return $general_tab_fields;
+		}
+
+		/**
+		 * Inject TEC specific setting fields into the display tab
+		 *
+		 * @param array $display_tab_fields Fields for the display settings tab
+		 */
+		public function display_settings_tab_fields( $display_tab_fields ) {
+			require_once $this->plugin_path . 'src/admin-views/tribe-options-display.php';
+
+			return $display_tab_fields;
+		}
+
+		/**
+		 * When TEC is activated, the Events top level menu item in the dashboard needs the post_type appended to it
+		 *
+		 * @param string $url Settings URL to filter
+		 */
+		public function tribe_settings_url( $url ) {
+			return add_query_arg( array( 'post_type' => self::POSTTYPE ), $url );
+		}
+
+		public function tribe_help_sidebar_top() {
+			$tec_info = wp_remote_get(
+			/**
+			 * Filter the tribe info API url
+			 *
+			 * @param string $url
+			 */
+			apply_filters( 'tribe_help_tab_api_info_url', Tribe__Events__Main::INFO_API_URL ), array(
+					'timeout' => 15, //seconds
+					'headers' => array( 'Accept' => 'application/json' ),
+				)
+			);
+			if ( ! is_wp_error( $tec_info ) ) {
+				$tec_info = $tec_info['body'];
+				$tec_info = unserialize( $tec_info );
+				if ( isset( $tec_info['rating'] ) ) {
+					$rating = $tec_info['rating'];
+				}
+				if ( isset( $tec_info['num_ratings'] ) ) {
+					$num_rating = $tec_info['num_ratings'];
+				}
+				if ( isset( $tec_info['requires'] ) ) {
+					$requires = $tec_info['requires'];
+				}
+				if ( isset( $tec_info['version'] ) ) {
+					$version = $tec_info['version'];
+				}
+				$total_downloads = ( isset( $tec_info['total_downloads'] ) ) ? number_format( $tec_info['total_downloads'] ) : _x( 'n/a', 'not available', 'tribe-common' );
+				$up_to_date      = ( isset( $tec_info['version'] ) && version_compare( Tribe__Events__Main::VERSION, $tec_info['version'], '<' ) ) ? __( 'You need to upgrade!', 'tribe-common' ) : __( 'You are up to date!', 'tribe-common' );
+			}
+
+			?>
+			<div id="tribe-help-plugin-info">
+				<h3><?php esc_html_e( 'The Events Calendar', 'tribe-common' ); ?></h3>
```

Comment:
Here and a few other places we're using the `tribe-common` textdomain - is that the right choice here, where we're not inside the common libs module?